src/main/java/com/mparticle/kits/IterableApi.java

```diff
@@ -0,0 +1,33 @@
+package com.mparticle.kits;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Created by David Truong dt@iterable.com.
+ */
+
+public class IterableApi {
+    private static Pattern deeplinkPattern = Pattern.compile("/a/[A-Za-z0-9]+");
```

Comment:
this seems like it's going to match everything - should it be matching only iterable urls?